lib/tasks/copyright.rake

```diff
@@ -119,6 +130,12 @@ namespace :copyright do
       file_content = File.read(file_name)
       if file_content.match(regexp)
         file_content.gsub!(regexp, "\\k<shebang>\\k<additional>#{copyright}")
+      elsif options[:create]
+        if file_content.include?("OpenProject is a fork of ChiliProject")
+          next puts "#{file_name} does not match regexp, but seems to have a copyright header!"
```

Comment:
Rubocop is ok with this? `next` accepts an argument, but it is not needed here, so I suggest splitting into two lines

Reply:
Rubocop didn't complain, but I can safely multiline this :D

I think Rubocop doesn't care, because `puts` is essentially just a method call and it returns nil.